src/ansi.rs

```diff
@@ -668,6 +668,9 @@ pub trait Handler {
 
     /// Set hyperlink.
     fn set_hyperlink(&mut self, _: Option<Hyperlink>) {}
+
+    /// Set mouse cursor.
+    fn set_mouse_cursor(&mut self, _: String) {}
```

Comment:
How are consumers supposed to know what this string means? There isn't even any documentation for it.

At the very least we should document the possible fields, or provide an enum with all possible values.

Reply:
I've updated the docs.

We could enforce the naming with the `cursor-icon` if it matters.

Reply:
Hm, github just bugged, because the git push said that everything up to date, but the commit wasn't matching the one on github, ammend + force push resolved though.

Reply:
I had the same thing happen to me the other day, it's really strange.